app/javascript/packs/banner_preview.jsx

```diff
@@ -78,8 +78,8 @@ export default class BannerPreview extends React.Component {
         currentDonation={0}
         balance={25.0}
         currentAmount={0}
-        bgImage={this.props.backgroundImage}
-        logo={this.props.logoImage}
+        bgImage={this.props.bannerImage.url}
```

Comment:
What's the purpose of this change?

Reply:
In the editor I consolidated `logo` and `logoData` into one variable `logo`. On the preview side we get the url via `this.props.logo.url`. Same for bannerImage.